n3fit/src/evolven3fit/evolve.py

```diff
@@ -0,0 +1,204 @@
+import pathlib
+import logging
+import numpy as np
+import yaml
+import sys
+
+from ekobox import genpdf, gen_info
+from ekomark import apply
+from eko import basis_rotation as br
+from eko import output
+
+from . import utils, eko_utils
+
+
+log = logging.getLogger(__name__)
+
+def evolve_fit(
+    conf_folder,
+    q_fin,
+    q_points,
+    op_card_dict,
+    t_card_dict,
+    eko_path=None,
+    dump_eko=None,
+):
+    """
+    Evolves all the fitted replica in conf_folder/nnfit
+
+    Parameters
+    ----------
+
+        conf_folder: str or pathlib.Path
+            path to the folder containing the fit
+        q_fin: float
+            final point of the q_grid
+        q_points: int
+            number of points in the q_grid
+        op_card_dict: dict
+            user settings for the op_card
+        t_card_dict: dict
+            user settings for the t_card
+        eko_path: str or pathlib.Path
+            path where the eko is stored (if None the eko will be
+            recomputed)
+        dump_eko: str or pathlib.Path
+            path where the eko is dumped (if None the eko won't be
+            stored)
+    """
+    log_file = pathlib.Path(conf_folder) / "evolven3fit.log"
+    if log_file.exists():
+        raise SystemError("Log file already exists: evolven3fit has already been run?")
```

Comment:
```suggestion
        raise FileExistsError(f"Log file already exists: {log_file} evolven3fit has already been run?")
```

Maybe `--force` flag is needed so that we don't have to manually remove the log file to re-evolve.